### PR TITLE
OBSDOCS-1178: Release notes for the Distributed Tracing 3.2.2 patch

### PR DESCRIPTION
**Pull request by max-cx**
:warning: Note for the reviewers: This PR reorganizes the currently published content and adjusts metadata for this purpose. The new files in this PR are just renamed existing files with moved and reused (copied and pasted) content from the deleted files. Here are the lines that actually have new content for the new patch release:

- https://github.com/openshift/openshift-docs/pull/78548#discussion_r1669465849
- https://github.com/openshift/openshift-docs/pull/78548#discussion_r1670247130
- https://github.com/openshift/openshift-docs/pull/78548#discussion_r1669465911
- https://github.com/openshift/openshift-docs/pull/78548#discussion_r1670248777

Version(s):

4.12, 4.13, 4.14, 4.15, 4.16, 4.17

Issue:

https://issues.redhat.com/browse/OBSDOCS-1178

Link to docs preview:

- https://78548--ocpdocs-pr.netlify.app/openshift-enterprise/latest/observability/distr_tracing/distr-tracing-rn.html
- https://78548--ocpdocs-pr.netlify.app/openshift-enterprise/latest/observability/otel/otel-rn.html

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: observability/distr_tracing/distr_tracing_rn/distr-tracing-rn-past-releases.adoc [hunk 1]
```diff
@@ -10,6 +10,133 @@ include::modules/distr-tracing-product-overview.adoc[leveloffset=+1]
...
+
+* Before this update, ... . (link:https://issues.redhat.com/browse/TRACING-????[TRACING-????])
+////
+
+[id="distributed-tracing-rn_3-2-1_tempo-release-notes_known-issues_{context}"]
+==== Known issues
+
+There is currently a known issue:
+
+* Currently, the {TempoShortName} fails on the IBM Z (`s390x`) architecture. (link:https://issues.redhat.com/browse/TRACING-3545[TRACING-3545])
```
[the remark this answers is not in the document]
Comment (max-cx) [reply observability/distr_tracing/distr_tracing_rn/distr-tracing-rn-past-releases.adoc:77]: Fixed.

File: observability/distr_tracing/distr_tracing_rn/distr-tracing-rn-past-releases.adoc [hunk 2]
```diff
@@ -10,6 +10,133 @@ include::modules/distr-tracing-product-overview.adoc[leveloffset=+1]
...
+
+////
+[id="distributed-tracing-rn_3-2-1_tempo-release-notes_technology-preview-features_{context}"]
+==== Technology Preview features
+
+This update introduces the following Technology Preview feature:
+
+* Support for the Tempo monolithic deployment.
+
+:FeatureName: The Tempo monolithic deployment
```
[the remark this answers is not in the document]
Comment (max-cx) [reply observability/distr_tracing/distr_tracing_rn/distr-tracing-rn-past-releases.adoc:36]: N/A

File: observability/distr_tracing/distr_tracing_rn/distr-tracing-rn-past-releases.adoc [hunk 3]
```diff
@@ -10,6 +10,133 @@ include::modules/distr-tracing-product-overview.adoc[leveloffset=+1]
...
+{JaegerName} 3.2.1 is supported for use with the {es-op} 5.6, 5.7, and 5.8.
+////
+
+////
+[id="distributed-tracing-rn_3-2-1_jaeger-release-notes_deprecated-functionality_{context}"]
+==== Deprecated functionality
+
+In the {DTProductName} 3.2.1, Jaeger and support for Elasticsearch remain deprecated, and both are planned to be removed in a future release.
+Red Hat will provide support for these components and fixes for CVEs and bugs with critical and higher severity during the current release lifecycle, but these components will no longer receive feature enhancements.
+The {TempoOperator} and the {OTELName} are the preferred Operators for distributed tracing collection and storage.
```
[the remark this answers is not in the document]
Comment (max-cx) [reply observability/distr_tracing/distr_tracing_rn/distr-tracing-rn-past-releases.adoc:103]: N/A

File: observability/distr_tracing/distr_tracing_rn/distr-tracing-rn-past-releases.adoc [hunk 4]
```diff
@@ -10,6 +10,133 @@ include::modules/distr-tracing-product-overview.adoc[leveloffset=+1]
...
+Users must adopt the OpenTelemetry and Tempo distributed tracing stack because it is the stack to be enhanced going forward.
```
[the remark this answers is not in the document]
Comment (max-cx) [reply observability/distr_tracing/distr_tracing_rn/distr-tracing-rn-past-releases.adoc:104]: N/A

File: observability/distr_tracing/distr_tracing_rn/distr-tracing-rn.adoc [hunk 5]
```diff
@@ -10,6 +10,133 @@ include::modules/distr-tracing-product-overview.adoc[leveloffset=+1]
...
+
+The {TempoName} is provided through the {TempoOperator}.
+
+////
+[id="distributed-tracing-rn_3-2-1_tempo-release-notes_technology-preview-features_{context}"]
+==== Technology Preview features
+
+This update introduces the following Technology Preview feature:
+
+* Support for the Tempo monolithic deployment.
```
[the remark this answers is not in the document]
Comment (max-cx) [reply observability/distr_tracing/distr_tracing_rn/distr-tracing-rn.adoc]: N/A

File: observability/distr_tracing/distr_tracing_rn/distr-tracing-rn.adoc [hunk 6]
```diff
@@ -10,6 +10,133 @@ include::modules/distr-tracing-product-overview.adoc[leveloffset=+1]
...
+
+:FeatureName: The Tempo monolithic deployment
```
[the remark this answers is not in the document]
Comment (max-cx) [reply observability/distr_tracing/distr_tracing_rn/distr-tracing-rn.adoc]: N/A

File: observability/distr_tracing/distr_tracing_rn/distr-tracing-rn.adoc [hunk 7]
```diff
@@ -10,6 +10,133 @@ include::modules/distr-tracing-product-overview.adoc[leveloffset=+1]
...
+{JaegerName} 3.2.1 is supported for use with the {es-op} 5.6, 5.7, and 5.8.
+////
+
+////
+[id="distributed-tracing-rn_3-2-1_jaeger-release-notes_deprecated-functionality_{context}"]
+==== Deprecated functionality
+
+In the {DTProductName} 3.2.1, Jaeger and support for Elasticsearch remain deprecated, and both are planned to be removed in a future release.
+Red Hat will provide support for these components and fixes for CVEs and bugs with critical and higher severity during the current release lifecycle, but these components will no longer receive feature enhancements.
+The {TempoOperator} and the {OTELName} are the preferred Operators for distributed tracing collection and storage.
```
[the remark this answers is not in the document]
Comment (max-cx) [reply observability/distr_tracing/distr_tracing_rn/distr-tracing-rn.adoc]: N/A

File: observability/distr_tracing/distr-tracing-rn.adoc [hunk 8]
```diff
@@ -10,6 +10,133 @@ include::modules/distr-tracing-product-overview.adoc[leveloffset=+1]
...
+{JaegerName} 3.2.1 is supported for use with the {es-op} 5.6, 5.7, and 5.8.
+////
+
+////
+[id="distributed-tracing-rn_3-2-1_jaeger-release-notes_deprecated-functionality_{context}"]
+==== Deprecated functionality
+
+In the {DTProductName} 3.2.1, Jaeger and support for Elasticsearch remain deprecated, and both are planned to be removed in a future release.
+Red Hat will provide support for these components and fixes for CVEs and bugs with critical and higher severity during the current release lifecycle, but these components will no longer receive feature enhancements.
+The {TempoOperator} and the {OTELName} are the preferred Operators for distributed tracing collection and storage.
```
[the remark this answers is not in the document]
Comment (max-cx) [reply observability/distr_tracing/distr-tracing-rn.adoc:229]: N/A

File: observability/distr_tracing/distr-tracing-rn.adoc [hunk 9]
```diff
@@ -10,6 +10,133 @@ include::modules/distr-tracing-product-overview.adoc[leveloffset=+1]
...
+Users must adopt the OpenTelemetry and Tempo distributed tracing stack because it is the stack to be enhanced going forward.
```
[the remark this answers is not in the document]
Comment (max-cx) [reply observability/distr_tracing/distr-tracing-rn.adoc:230]: N/A

File: observability/otel/otel-rn.adoc [hunk 10]
```diff
@@ -1,26 +1,156 @@
...
+
+The {OTELName} is provided through the {OTELOperator}.
+
+[id="otel_3-2-2_cves_{context}"]
+=== CVEs
+
+This release fixes the following CVEs:
+
+* link:https://access.redhat.com/security/cve/CVE-2023-2953/[CVE-2023-2953]
+* link:https://access.redhat.com/security/cve/CVE-2024-28182[CVE-2024-28182]
```
Comment (max-cx) [inline observability/otel/otel-rn.adoc:22]: New content.

File: observability/distr_tracing/distr-tracing-rn.adoc [hunk 11]
```diff
@@ -1,33 +1,289 @@
...
+
+This release of the {DTProductName} includes the {TempoName} and the deprecated {JaegerName}.
+
+[id="distr-tracing_3-2-2_cves_{context}"]
+=== CVEs
+
+This release fixes the following CVEs:
+
+* link:https://access.redhat.com/security/cve/CVE-2023-2953/[CVE-2023-2953]
+* link:https://access.redhat.com/security/cve/CVE-2024-28182[CVE-2024-28182]
```
Comment (max-cx) [inline observability/distr_tracing/distr-tracing-rn.adoc:24]: New content.

File: observability/distr_tracing/distr-tracing-rn.adoc [hunk 12]
```diff
@@ -1,33 +1,287 @@
...
+
+In {TempoName} 3.2.2, the FEATURE has been removed. Bug fixes and support are provided only through the end of the 3.? lifecycle. As an alternative to the FEATURE for USE CASE, you can use the ALTERNATIVE instead.
+////
+
+[id="distr-tracing_3-2-2_tempo-release-notes_bug-fixes_{context}"]
+==== Bug fixes
+
+This update introduces the following bug fix:
+
+* Before this update, secrets were perpetually created on {product-title} 4.16 every second. With this update, this issue is fixed. (link:https://issues.redhat.com/browse/TRACING-4434/[TRACING-4434])
```
Comment (max-cx) [inline observability/distr_tracing/distr-tracing-rn.adoc:71]: New content.

File: observability/otel/otel-rn.adoc [hunk 13]
```diff
@@ -1,26 +1,154 @@
...
+
+In the {OTELName} 3.2.2, the FEATURE has been removed. Bug fixes and support are provided only through the end of the 3.? lifecycle. As an alternative to the FEATURE for USE CASE, you can use the ALTERNATIVE instead.
+////
+
+[id="otel_3-2-2_bug-fixes_{context}"]
+=== Bug fixes
+
+This update introduces the following bug fix:
+
+* Before this update, secrets were perpetually created on {product-title} 4.16 every second. With this update, this issue is fixed. (link:https://issues.redhat.com/browse/TRACING-4435/[TRACING-4435])
```
Comment (max-cx) [inline observability/otel/otel-rn.adoc:64]: New content.

**Review by pavolloffay — APPROVED** [review]

File: observability/distr_tracing/distr-tracing-rn.adoc [hunk 14]
```diff
@@ -1,33 +1,287 @@
...
+
+In {TempoName} 3.2.2, the FEATURE has been removed. Bug fixes and support are provided only through the end of the 3.? lifecycle. As an alternative to the FEATURE for USE CASE, you can use the ALTERNATIVE instead.
+////
+
+[id="distr-tracing_3-2-2_tempo-release-notes_bug-fixes_{context}"]
+==== Bug fixes
+
+This update introduces the following bug fix:
+
+* Before this update, secrets were perpetually generated on {product-title} 4.16 because the operator tried to reconcile a new `openshift.io/internal-registry-pull-secret-ref` annotation for service accounts, causing a loop. With this update, the operator ignores this new annotation. (link:https://issues.redhat.com/browse/TRACING-4434/[TRACING-4434])
```
Comment (max-cx) [inline observability/distr_tracing/distr-tracing-rn.adoc:71]: New content.

File: observability/otel/otel-rn.adoc [hunk 15]
```diff
@@ -1,26 +1,154 @@
...
+
+In the {OTELName} 3.2.2, the FEATURE has been removed. Bug fixes and support are provided only through the end of the 3.? lifecycle. As an alternative to the FEATURE for USE CASE, you can use the ALTERNATIVE instead.
+////
+
+[id="otel_3-2-2_bug-fixes_{context}"]
+=== Bug fixes
+
+This update introduces the following bug fix:
+
+* Before this update, secrets were perpetually generated on {product-title} 4.16 because the operator tried to reconcile a new `openshift.io/internal-registry-pull-secret-ref` annotation for service accounts, causing a loop. With this update, the operator ignores this new annotation. (link:https://issues.redhat.com/browse/TRACING-4435/[TRACING-4435])
```
Comment (max-cx) [inline observability/otel/otel-rn.adoc:64]: New content.

**Review by sheriff-rh — APPROVED** [review]
Hi Max, thanks for your note at the top regarding the new content, that was super helpful. 

These changes look sane to me and are in-line with our docs guidance, so I will go ahead and merge to 4.12+.

**Pull request merged**